### ISSUE TITLE
Passing uninitialized address-type variable to format() results in a runtime exception

### ISSUE THREAD

**Issue by BorisI** [everx-labs/TVM-Solidity-Compiler#168]
```TvmSolidity
pragma ton-solidity >= 0.74.0;
contract fma {
    function foo() external pure returns (string s) {
        address v;
        s.append(format("{}\n", v));
    }
}
```

```shell
$ ever-cli -c etc/fma.conf debug run -m foo; tail -n11 trace.log  | head -n7

52 1893 18 NEWC fma.sol:6
53 1911 18 NULL fma.sol:6
54 1939 28 TUPLE 2 fma.sol:6
55 1961 22 PUSHSLICE x2_ fma.sol:6
56 2087 126 CALLREF stdlib.sol:215
57 2113 26 REWRITESTDADDR stdlib.sol:214
58 2163 0 UNHANDLED EXCEPTION: VM Exception: 0 1 stdlib.sol:214
```

**Comment by IgorKoval** [comment]
Fixed in 0.77.0

**Comment by Subway2023** [comment]
@BorisI How do I execute the program, and how should the etc/fma.conf be configured?
